pkg/cloud-provider/cloudapi/aws/aws_crd_helpers.go

```diff
@@ -74,9 +74,20 @@ func ec2InstanceToInternalVirtualMachineObject(instance *ec2.Instance, namespace
 	cloudID := *instance.InstanceId
 	cloudNetwork := *instance.VpcId
 
-	return utils.GenerateInternalVirtualMachineObject(cloudID, strings.ToLower(cloudName), strings.ToLower(cloudID), strings.ToLower(region),
-		namespace, strings.ToLower(cloudNetwork), cloudNetwork, runtimev1alpha1.VMState(*instance.State.Name), tags, networkInterfaces,
-		providerType, account)
+	var vpcName string
```

Comment:
create a new function for this

Reply:
Done, created a function for extracting vpc name from vpcs map.